mjit_c.rb

```diff
@@ -13,6 +13,30 @@ def SHAPE_FLAG_SHIFT
       Primitive.cexpr! 'UINT2NUM(SHAPE_FLAG_SHIFT)'
     end
 
+    def SHAPE_ROOT
+      Primitive.cexpr! 'UINT2NUM(SHAPE_ROOT)'
+    end
+
+    def SHAPE_IVAR
+      Primitive.cexpr! 'UINT2NUM(SHAPE_IVAR)'
+    end
+
+    def SHAPE_FROZEN
+      Primitive.cexpr! 'UINT2NUM(SHAPE_FROZEN)'
+    end
+
+    def SHAPE_CAPACITY_CHANGE
+      Primitive.cexpr! 'UINT2NUM(SHAPE_CAPACITY_CHANGE)'
+    end
+
+    def SHAPE_IVAR_UNDEF
+      Primitive.cexpr! 'UINT2NUM(SHAPE_IVAR_UNDEF)'
+    end
+
+    def SHAPE_INITIAL_CAPACITY
+      Primitive.cexpr! 'UINT2NUM(SHAPE_INITIAL_CAPACITY)'
+    end
```

Comment:
This section is manually maintained, but those simple definitions could be semi-automatically defined through mjit-bindgen https://github.com/ruby/ruby/commit/a8537aa6a5ec6da5261e727943ba267bc422e594. I'd like to eventually generate everything in this file through mjit-bindgen, so it'd be nice if we can use it whenever possible.